### PR TITLE
2.0 editors draft

### PR DESCRIPTION
**Pull request by ldodds**
Started drafting the 2.0 specification.

* Revised all the core models to align with new validation rules and clarified formats of properties
* Removed Activity Opportunity
* Improved extension section to include example of custom context
* Various minor text improvements

Still to do here:

* [x] add tables for subsidiary types, e.g. `GeoCoordinates`
* [x] Fix other bugs
* [x] check that conformance criteria in spreadsheet are clear in spec
* [x] Subtypes of Event and additional rules
* [ ] Add section for url templates 
* [ ] publish initial draft

Still to do elsewhere:

* [x] Add Brand to validation spreadsheet
* [ ] Revise ns to refer to `oa:concept` not `oa:concepts`
* [ ] Review data models to align with spreadsheet, add examples
* [x] File issue about multiple contexts and indicate conformance rules
* [ ] Add `oa:groupFacilityUse` to NS
* [ ] Renamed `individualFacilityUses` in NS
* [ ] Add `isAccessibleWithoutBooking` to NS
* [ ] Renamed `PublicToilets` and `ChangingRooms` in NS
* [ ] Add changes from Schema, e.g. `idTemplate`, `sessionInstanceType`, etc.

### CODE REVIEW

File: EditorsDraft/index.html [hunk 1]
```diff
@@ -1614,8 +1648,9 @@
       "type": "PostalAddress",
       "streetAddress": "North Parade Road",
       "addressLocality": "Bath",
-      "region": "Somerset",
-      "postalCode": "BA2 4ET"
+      "addressRegion": "Somerset",
+      "postalCode": "BA2 4ET",
+      "addressCountry": "UK"
```
Comment (nickevansuk) [inline EditorsDraft/index.html]: Should be "GB", the two-letter [ISO 3166-1 alpha-2 country code](http://en.wikipedia.org/wiki/ISO_3166-1).

File: EditorsDraft/index.html [hunk 2]
```diff
@@ -1633,8 +1668,9 @@
...
```
Comment (nickevansuk) [inline EditorsDraft/index.html]: Should be "GB", the two-letter [ISO 3166-1 alpha-2 country code](http://en.wikipedia.org/wiki/ISO_3166-1).

File: EditorsDraft/index.html [hunk 3]
```diff
@@ -1375,17 +1424,23 @@
      "repeatFrequency": "P1W",
      "byDay": "http://schema.org/Wednesday",
      "startTime": "19:00Z",
-     "endTime": "20:00Z"
+     "endTime": "20:00Z",
```
Comment (nickevansuk) [inline EditorsDraft/index.html]: This is incorrect and should be resolved: see https://github.com/openactive/data-model-validator/issues/99 - the timezone component doesn't make any sense in this context. A Yoga session would be Tuesday 7pm in the winter, and Tuesday 8pm in the summer.

File: EditorsDraft/index.html [hunk 4]
```diff
@@ -1645,6 +1681,73 @@
...
+<pre class="example" title="An Address">
+<code>
+{
+  "@context": "https://www.openactive.io/ns/oa.jsonld",
+  "type": "PostalAddress",
+  "streetAddress": "North Parade Road",
+  "addressLocality": "Bath",
+  "addressRegion": "Somerset",
+  "postalCode": "BA2 4ET",
+  "addressCountry": "UK"
```
Comment (nickevansuk) [inline EditorsDraft/index.html]: Should be "GB", the two-letter ISO 3166-1 alpha-2 country code.

File: EditorsDraft/index.html [hunk 5]
```diff
@@ -2336,55 +2546,163 @@
...
+
+[[JSON-LD]] allows data to be [published with reference to multiple contexts](https://www.w3.org/TR/json-ld/#advanced-context-usage). 
+
+Assuming that the previous JSON-LD context was published to `https://example.org/custom.jsonld`, then 
+the following example shows how to use this extension:
+
+<pre class="example" title="Example of a custom context"><code>
+{
+  "@context": [
+    "https://www.openactive.io/ns/oa.jsonld",
```
Comment (nickevansuk) [inline EditorsDraft/index.html]: Note - the validator currently fails for arrays in context

File: EditorsDraft/index.html [hunk 6]
```diff
@@ -1375,17 +1424,23 @@
      "repeatFrequency": "P1W",
      "byDay": "http://schema.org/Wednesday",
      "startTime": "19:00Z",
-     "endTime": "20:00Z"
+     "endTime": "20:00Z",
+     "duration": "P1H"
```
Comment (petewalker) [inline EditorsDraft/index.html]: This is an invalid duration - should be `PT1H`

File: EditorsDraft/index.html [hunk 7]
```diff
@@ -1715,14 +1821,15 @@
       "type": "PostalAddress",
       "streetAddress": "North Parade Road",
       "addressLocality": "Bath",
-      "region": "Somerset",
-      "postalCode": "BA2 4ET"
+      "addressRegion": "Somerset",
+      "postalCode": "BA2 4ET",
+      "addressCountry": "UK"
```
Comment (petewalker) [inline EditorsDraft/index.html]: Should be "GB" as per @nickevansuk comments

File: EditorsDraft/index.html (hunk 5 above)
Comment (petewalker) [reply EditorsDraft/index.html]: Not any more..!

File: EditorsDraft/index.html [hunk 8]
```diff
@@ -1743,11 +1830,70 @@
...
+      "addressLocality": "Bath",
+      "addressRegion": "Somerset",
+      "postalCode": "BA2 4ET",
+      "addressCountry": "GB"
+  },
+  "telephone": "+44 (0)1225 486905",
+  "geo": {
+    "type": "GeoCoordinates",
+    "latitude": "51.3816123",
+    "longitude": "-2.3544367"
```
Comment (nickevansuk) [inline EditorsDraft/index.html]: Should be number

**Pull request merged**